src/value/mod.rs

```diff
@@ -299,6 +299,9 @@ impl WasmValue for Value {
                     .ok_or_else(|| ValueError::InvalidValue(format!("unknown flag `{name}`")))
             })
             .collect::<Result<Vec<_>, ValueError>>()?;
+        // Flags values don't logically contain an ordering of the flags. Sort
+        // the flags values so that equivalent flags values compare equal.
+        flags.sort();
```

Comment:
Wasmtime lifts flags into the order they were declared in. If we care about equality outside of this crate we should probably sort by `flag_names` index here instead. (or fix Wasmtime's `PartialEq` impl)

Reply:
Oh woops, I misunderstood which impl this was.